test/mysql-config/my.cnf

```diff
@@ -0,0 +1,2 @@
+[mysqld]
+sql_mode = "NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION"
```

Comment:
Now that I see why it's done, the `only_full_group_by` is the default in MySQL 5.7 (which is probably why it's happening in Docker). We should probably fix these bugs rather than hiding them.